src/CMakeLists.txt

```diff
@@ -188,6 +188,7 @@ SET(LMMS_REQUIRED_LIBS ${LMMS_REQUIRED_LIBS}
 	${SNDFILE_LIBRARIES}
 	${FFTW3F_LIBRARIES}
 	${EXTRA_LIBRARIES}
+	hiir
```

Comment:
I'd rather this be added as a dependency when and where it's needed, rather than having the core depend on it right now even though there's no code using it.